assets/composer.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
We can still put this in /src it just won't do anything. But let's move it to /src/custom, or something that keeps it close to the command files and doesn't make it seem like it's the _project_ composer.json